src/nvapi_d3d12.cpp

```diff
@@ -236,6 +236,7 @@ extern "C" {
                 pGraphicsCaps->minorSMVersion = 0;
                 break;
             case NV_GPU_ARCHITECTURE_GM200:
```

Comment:
Accoording to https://en.wikipedia.org/wiki/Maxwell_(microarchitecture) we should correct this and report sm52 for Maxwell Gen2 and sm50 for Maxwell Gen2.

Reply:
See info about `Series_*` too.

Reply:
Well, `*_series`.
https://en.wikipedia.org/wiki/GeForce_700_series
https://en.wikipedia.org/wiki/GeForce_800M_series
https://en.wikipedia.org/wiki/GeForce_900_series

All contains info about `numbers` for Maxwell Gen1.
Just in case.